extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/ContainerImageConfig.java

```diff
@@ -69,6 +69,13 @@
     @ConfigItem
     public boolean push;
 
+    /**
+     * The name of the container image extension to use (e.g. docker, jib, s2i).
+     * The option will be used in case multiple extensions are present.
```

Comment:
I think we should also update the container image documentation to mention this.

Reply:
Do you plan to add this in a next commit?

Reply:
Just added the extra doc.
Also updated the error message that is displayed when there are conflicitng container image extension to inform the user about the `quarkus.container-image.builer` property.